pkg/http/server.go

```diff
@@ -128,11 +132,17 @@ func (s *server) StartNonBlocking() error {
 		return errors.New("could not listen on any endpoint")
 	}
 
+	// Create a handler with support for HTTP/2 Cleartext
+	var handler http.Handler = r
+	if !utils.IsTruthy(os.Getenv("DAPR_HTTP_DISABLE_H2C")) {
+		handler = h2c.NewHandler(r, &http2.Server{})
```

Comment:
qq can TLS still be set with h2c handler?

Reply:
H2C is HTTP/2 Cleartext so it cannot use TLS by definition :)

However our server (now) does support HTTP/2 automatically when TLS is enabled. But that only work when using HTTPS.